src/seeder.js

```diff
@@ -119,8 +138,16 @@ export default class Seeder {
      * Find all the items in the DB and start seeding them again
      */
     async _seedExistingItems() {
+        const keys = []
         const stream = this.db.createReadStream()
-        stream.on('data', async (data) => await this._beginSeeding(data.key))
+        stream.on('data', (data) => keys.push(data.key))
```

Comment:
read streams are also iterable (I can't use that in Bitkit because reasons).

But this can be 
```js
for await (let data of this.db.createReadStream()) {
    keys.push(data.key)
}
```

Reply:
TypeError: this.db.createReadStream is not a function or its return value is not iterable

Seems it is not iterable...

Reply:
asynIterable ... it sure is https://github.com/hypercore-protocol/hyperbee/blob/1e4eac02fbf57e72ccd0be3d3c2bac666e15dfbe/test/all.js#L312